.github/workflows/serde_compat.yml

```diff
@@ -0,0 +1,27 @@
+name: Java SerDe Compatibility Test
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
do we want a manual trigger option for if we're trying to update serialization in a sketch, without needing to open a PR?

Reply:
perhaps. I will add the manual trigger